src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -464,15 +465,36 @@ public Task GetMigratedServerUrls()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/smuggler/migrator-path", "GET", AuthorizationStatus.Operator)]
+        public Task GetConfigurationMigratorPath()
```

Comment:
Isn't this more like a StudioTaskHandler endpoint? This is used purely for Studio, right?

Is Operator fine (I think it is). Who can do migration?

Reply:
Yes, Studio only. I'll move it.
Migration can be done by Operator.